### ISSUE TITLE
What is the procedure for keeping this documentation in sync with package documentation?

### ISSUE THREAD

**Issue by zygmuntszpak** [JuliaImages/juliaimages.github.io#26]
I've extended some of the documentation by modifying various docstrings of ImageFiltering.jl and these have been merged in the ImageFiltering repository. However, the new documentation is not reflected here. How does one go about rebuilding the documentation so that the latest docstrings appear here as well?

**Comment by Evizero** [comment]
I'll look into it. Thanks for bringing this up

**Comment by Evizero** [comment]
looks to me like travis here triggers the doc build. meaning to update the docs one needs to trigger travis on this repository. I just merged #25 so this should rebuild the docs. I.e. in the future someone needs to trigger travis manually if the docs are to be rebuilt

**Comment by zygmuntszpak** [comment]
Thanks for looking into this. This seems not to have worked. I just had a look at the latest documentation and the function reference for the image filtering package is still the old version.

**Comment by Evizero** [comment]
Right. I just looked into it. At first I thought it might be the failed doctests (logs: https://travis-ci.org/JuliaImages/juliaimages.github.io/builds/353799257 ), but then I realised the cause is much simpler.

The way this documentation website works is it pulls all the docs from the latest **tagged** version. which means that I (or someone) has to tag a new release before they are included here.

I'll look into tagging a release
